sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -63,20 +58,39 @@ add_sycl_plugin(unified_runtime
     "pi2ur.cpp"
     "ur/ur.hpp"
     "ur/ur.cpp"
-    "ur/adapters/level_zero/ur_level_zero.hpp"
-    "ur/adapters/level_zero/ur_level_zero.cpp"
     "ur/usm_allocator.hpp"
```

Comment:
I think ur.*pp and usm_allocator.*pp files should not be needed here. They should be used to build ur_adapter_level_zero

Reply:
thanks will fix